sdk/playwrighttesting/microsoft-playwright-testing/src/reporter/mptReporter.ts

```diff
@@ -394,6 +395,18 @@ class MPTReporter implements Reporter {
       );
       this.isRegionValid = false;
     }
+    if (
+      this.envVariables.accountId &&
+      !ReporterUtils.validateWorkspace(this.envVariables.accountId)
+    ) {
+      this.isSameWorkspace = false;
+      const errorMessage = Constants.workspaceMismatchError.message;
```

Comment:
No need to check for keys here since this is a 1 time operation. You can directly insert it in the informationMessage array.